Automattic/wp-calypso
# Pricing Grid: Styling the header

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71407.
* This PR adds styling changes for the header section (above the CTA buttons) to look like in Figma.

<img width="1528" alt="Screenshot 2022-12-26 at 12 10 21 PM" src="https://user-images.githubusercontent.com/1269602/209513098-d8423f1e-4241-42e7-a225-06cff255f8f3.png">


#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.

#### Known Issues

* The Premium plan does not have the styling to highlight it as Popular. This is fixed in a subsequent PR.
* Clicking the CTAs on the Free and Enterprise plan don't work and are fixed in a subsequent PR.

#### Testing Instructions



* Visit `/start/onboarding-2023-pricing-grid` and verify that the header section (screenshot shown in the description) matches the design in Figma. In particular, pay attention to the following:
    * Fonts - font sizes, font weights, font family, color and so on.
    * Margins and paddings
* Verify that the UI matches in both monthly and annual plan toggles.
* Verify that the price display looks good in different currencies - e.g. `IDR` and `JPY` . In currencies like `INR` and `MXN`, we offer a first year discount, so there's a strike-out text on the price. The currencies we support are in 2e9ad-pb.
* Verify that the production `onboarding` flow at `/start` is unaffected by UI changes.
